### DISCUSSION TITLE
how to define the order of markdowns files?

### DISCUSSION THREAD

**Discussion by JumpWill** [mkdocs/mkdocs#3790] [Q&A] [answered]
i have markdown files like
    chapter-1
    chapter-2
    ...
    chapter-11
    chapter-12,
i prefer the order show in web is 
    chapter-1
    chapter-2
    ...
    chapter-11
    chapter-12,
the order show in web is 
      chapter-1
      chapter-11
      chapter-12
      ...
      chapter-2,

i do not wanna define nav in mkdocs, because  i do not know  if i have to define all of the markdown files,

how to define the order  without specifying the 'nav' attribute?
or 
can i only define  the specific directory nav in  mkdocs.yaml

**Comment by pawamoy** [comment, marked as answer]
I don't think there's any locale that will allow that (that you could have used as value of the `LC_ALL` environment variable).

So you will have to define your `nav`, or use a plugin to define it partially:

- https://github.com/oprypin/mkdocs-literate-nav
- https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin
- more in the catalog: https://github.com/mkdocs/catalog/

You could also rename your files like `01-chapter-1` and define front-matter titles:

```md
---
title: Chapter 1
---

# Chapter 1

...markdown contents
```

...however you'd still have `01-chapter-1` in the URL I believe.
